packages/react-components/src/components/ControlBarButton.tsx

```diff
@@ -137,6 +144,8 @@ export const ControlBarButton = (props: ControlBarButtonProps): JSX.Element => {
         onRenderIcon={props.onRenderIcon ?? DefaultRenderIcon}
         ariaLabel={props.ariaLabel ?? tooltipContent ?? labelText}
         allowDisabledFocus={props.allowDisabledFocus ?? true}
+        aria-roledescription={'split button'}
```

Comment:
should this be being set here? (was this left in?)

Reply:
oops will fix it!